test/qt3testsBenchmark.ts

```diff
@@ -0,0 +1,263 @@
+import { Suite } from 'benchmark';
+
+import {
+	evaluateXPathToBoolean,
+	evaluateXPathToFirstNode,
+	evaluateXPathToNodes,
+	evaluateXPathToNumber,
+	evaluateXPathToString,
+	Language,
+	Document,
+} from 'fontoxpath';
+
+import { ALL_TESTS_QUERY, getAllTestSets, getArguments, getFile } from 'test-helpers/qt3TestsTools';
+import getPerformanceTests from 'test-helpers/getPerformanceTests';
+import { Node, Element } from 'slimdom';
+
+const suite: Suite = new Suite();
+
+function addTests(functions) {
+	Object.keys(functions).forEach((name) => {
+		suite.add(name, functions[name]);
+	});
+}
+
+function runTests() {
+	console.log('Tests are running...');
+	suite
+		// add listeners
+		.on('cycle', (event) => {
+			console.log(
+				`${event.target.name}, ${Math.round(event.target.hz * 1000) / 1000} op/sec,±${
```

Comment:
NIT: also add a space after the comma in `op/sec,±$` or have no spaces after any comma.
I'm actually ok with with either as it should not matter when saving it to a csv but I'd prefer to have it consistent.

Reply:
I go for a space after comma, therefore it looks better in the log